2020-12-21 18:49:19.682 | INFO     | __main__:<module>:6 - This is used to load data into RDS instance
2020-12-21 18:49:19.698 | INFO     | utils:load_tables:118 - found the load details of table
2020-12-21 18:53:37.066 | INFO     | __main__:<module>:6 - This is used to load data into RDS instance
2020-12-21 18:53:37.066 | INFO     | utils:load_tables:118 - found the load details of table
2020-12-21 18:53:37.081 | ERROR    | utils:wrapper:37 - Exception details are read_file() got an unexpected keyword argument 'file_type'
Traceback (most recent call last):

  File "main.py", line 13, in <module>
    data = read_file(file_name, file_type=file_extension)
           |         |                    -> 'csv'
           |         -> 'iris.csv'
           -> <function udf_exception.<locals>.wrapper at 0x000002AC6CE82F70>

> File "C:\Users\Sai Prashanth T S\PycharmProjects\Load_data_into_RDS\scripts\utils.py", line 33, in wrapper
    result = method1(*args, **kwargs)
             |        |       -> {'file_type': 'csv'}
             |        -> ('iris.csv',)
             -> <function read_file at 0x000002AC6CE82EE0>

TypeError: read_file() got an unexpected keyword argument 'file_type'
2020-12-21 18:53:55.421 | INFO     | __main__:<module>:6 - This is used to load data into RDS instance
2020-12-21 18:53:55.421 | INFO     | utils:load_tables:118 - found the load details of table
2020-12-21 18:53:55.437 | INFO     | utils:read_file:95 - read the file as pandas dataframe
2020-12-21 18:54:10.225 | INFO     | __main__:<module>:6 - This is used to load data into RDS instance
2020-12-21 18:54:10.225 | INFO     | utils:load_tables:118 - found the load details of table
2020-12-21 18:54:10.241 | INFO     | utils:read_file:95 - read the file as pandas dataframe
2020-12-21 18:54:10.241 | INFO     | __main__:<module>:14 -    sepal_length  sepal_width  petal_length  petal_width species
0           5.1          3.5           1.4          0.2  setosa
1           4.9          3.0           1.4          0.2  setosa
2           4.7          3.2           1.3          0.2  setosa
3           4.6          3.1           1.5          0.2  setosa
4           5.0          3.6           1.4          0.2  setosa
2020-12-21 19:05:36.850 | INFO     | __main__:<module>:39 - We are loading data to dev environment
2020-12-21 19:05:36.850 | INFO     | __main__:<module>:40 - Loading data process begins now
2020-12-21 19:05:36.850 | INFO     | __main__:run:8 - This is used to load data into RDS instance
2020-12-21 19:05:36.850 | INFO     | utils:load_tables:118 - found the load details of table
2020-12-21 19:05:36.850 | INFO     | __main__:run:16 - Name of the source file is iris.csv
2020-12-21 19:05:36.850 | INFO     | __main__:run:17 - Name of the target table is iris
2020-12-21 19:05:36.850 | INFO     | utils:read_file:95 - read the file as pandas dataframe
2020-12-21 19:05:36.865 | INFO     | __main__:run:21 - Sample data is shown below for your reference
2020-12-21 19:05:36.865 | INFO     | __main__:run:22 -    sepal_length  sepal_width  petal_length  petal_width species
0           5.1          3.5           1.4          0.2  setosa
1           4.9          3.0           1.4          0.2  setosa
2           4.7          3.2           1.3          0.2  setosa
3           4.6          3.1           1.5          0.2  setosa
4           5.0          3.6           1.4          0.2  setosa
2020-12-21 19:05:36.881 | INFO     | utils:wrapper:20 - Total_time taken to load the function 0:00:00.031243
2020-12-21 19:05:36.881 | INFO     | __main__:<module>:42 - Loading data process ends now
